namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/variables.tf

```diff
@@ -10,11 +10,11 @@ variable "kubernetes_cluster" {
 
 variable "application" {
   description = "Name of Application you are deploying"
-  default     = "Gov.UK Prototype Kit"
+  default     = "Approved Premises Prototype"
 }
 
 variable "namespace" {
-  default = "approved-premises-prototype"
+  default = "approved-premises-team"
```

Comment:
This should still be "approved-premises-prototype"
The variable that needed to be change was 

variable "team_name" {
  description = "The name of your development team"
  default     = "approved-premises-prototype" > "approved-premises-team"
}

Reply:
Gotcha. Will fix 👍